src/core/Akka/Dispatch/Mailbox.cs

```diff
@@ -375,8 +375,7 @@ private void ProcessMailbox(int left, long deadlineTicks)
         {
             while (ShouldProcessMessage())
             {
-                Envelope next;
-                if (!TryDequeue(out next)) return;
+                if (!TryDequeue(out var next)) return;
```

Comment:
Changed mailbox to use new `out var` syntax. Don't expect this to have any real performance impact.